[CONTENT] More non-romantic relationship interactions

howlidae opened the pull request:
## About The Pull Request

I've added over a 1000 interactions to the normal-interactions folder and its subfolders. Also added a few missing interaction types, removed some that had the chance to not make sense, and added some additional traits to interaction types for a larger pool.

I am open to any edits being made, any errors being corrected, and any lines being slashed. I intended this to just be a mod, so there is a strong chance that there are some things not quite fit for ClanGen!!

## Why This Is Good For ClanGen

The 'relationships' tab in events tends to become a bit repetitive. With how many people contribute to thoughts, patrols, and romantic interactions, I thought it'd be nice to contribute to normal interactions.

## Proof of Testing

A few examples:
![image](https://github.com/user-attachments/assets/465c6f12-d393-44cb-b2c4-fa38bf16b9d4)
![image](https://github.com/user-attachments/assets/7491d770-188e-4a9b-8838-3f62885c5212)
![image](https://github.com/user-attachments/assets/86f774a7-b04e-47a7-b8e5-367e92c53152)

## Changelog/Credits

added new interactions to normal-interactions

howlidae committed:
replaced admiration files

added interactions

howlidae committed:
replaced comfortable files

added interactions

howlidae committed:
replaced dislike files

added interactions

howlidae committed:
replaced jealousy files

added interactions

howlidae committed:
replaced platonic files

added interactions

howlidae committed:
replaced trust files

added interactions

howlidae committed:
replaced neutral file

added interactions

scribblecrumb commented:
Looks like you've got some tests failing, though I believe they're all being caused by broken jsons, specifically comfortable increase and dislike increase.  You can pop them into https://jsonlint.com/ to find the mistake quickly.

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/admiration/decrease.json:
```diff
@@ -1,152 +1,215 @@
+			"m_c notices that r_c overslept.",
+			"r_c tells m_c to smile more.",
+			"m_c had a funny dream about r_c and can't take {PRONOUN/r_c/object} seriously now.",
+			"r_c is talking to m_c, but {PRONOUN/m_c/subject} can't stop noticing the mud on {PRONOUN/r_c/poss} pelt."
```
```suggestion
			"r_c is talking to m_c, but {PRONOUN/m_c/subject} can't stop noticing the mud on r_c's pelt."

```
In this instance, if both cats had the same pronouns then it could be misinterpreted as m_c having mud on their pelt, rather than r_c.

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/admiration/decrease.json:
```diff
@@ -1,152 +1,215 @@
+			"r_c makes a snide remark about one of m_c's friends.",
+			"r_c confuses m_c in a conversation.",
+			"m_c and r_c disagree on something that should have an obvious answer.",
+			"r_c tells m_c to be happier. m_c explains it's not that simple.",
```
This might be a nice one to make trait constrained, perhaps even both cats could require specific traits.  It just feels like such a specific interaction.  You could have m_c be gloomy, lonesome, grumpy, or cold.  And then have r_c be confident, arrogant, shameless, flamboyant, childish, playful, or oblivious.

howlidae committed:
fix errors in increase.json

howlidae committed:
Update decrease.json

howlidae committed:
Update increase.json

howlidae committed:
fix errors in increase.json

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/admiration/decrease.json:
```diff
@@ -1,152 +1,215 @@
+			"leader"
+		],
+		"random_status_constraint": [
+			"elder",
```
I think elder should be removed from here.  A lot of interactions don't make sense for r_c as an elder.

howlidae commented:
Whoopsies!! I believe they should be fixed now. Looks like I made a couple of edits and forgot to test them. Thanks!

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/admiration/decrease.json:
```diff
@@ -1,152 +1,215 @@
+			"m_c notices that r_c has been avoiding {PRONOUN/r_c/poss} responsibilities lately.",
+			"m_c and r_c were supposed to be working together today, but they can never agree long enough to accomplish anything.",
+			"m_c is frustrated after finishing a task that r_c was supposed to do.",
+			"m_c pranked r_c during a patrol.",
+			"m_c is upset that r_c slept through patrol assignments.",
+			"m_c thinks that r_c acted disrespectfully during a patrol.",
+			"m_c noticed that r_c ignored an herb opportunity on a recent patrol.",
+			"After disagreeing on what to do during a recent patrol, m_c is frustrated with r_c."
```
You might consider moving some of these to the "admire_adult_de_med1" block.  Many of these aren't appropriate for elders or mediators to take part in.

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/admiration/decrease.json:
```diff
@@ -1,152 +1,215 @@
+			"r_c was supposed to be showing m_c a cool battle move but messed up so badly it made m_c cringe.",
+			"m_c noticed that r_c ignored an easy hunting opportunity on a recent patrol.",
+			"r_c froze up during a hunting patrol with m_c.",
+			"When m_c's border patrol encountered another clan, r_c acted like a fool.",
```
```suggestion
			"When m_c's border patrol encountered another Clan, r_c acted like a fool.",

```

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/admiration/decrease.json:
```diff
@@ -1,152 +1,215 @@
+			"m_c notices that r_c seems to disappear whenever difficult tasks are being discussed.",
+			"r_c was meant to take a piece of prey to someone sick, but m_c saw {PRONOUN/r_c/object} eat it.",
+			"r_c didn't finish an important chore that m_c assigned {PRONOUN/m_c/object}.",
+			"The Clan notices that m_c and r_c won't talk to each other after coming back from a patrol."
```
I think this should be moved to admire_adult_de_med1, as it's inappropriate for an m_c elder

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/admiration/decrease.json:
```diff
@@ -1,152 +1,215 @@
+			"m_c realizes r_c stole something from {PRONOUN/m_c/poss} nest.",
+			"m_c overhears r_c say something awful about a dead Clanmate.",
+			"m_c struggles to remember the last time r_c did something helpful.",
+			"When a bout of whitecough spreads around camp, m_c notices that r_c doesn't take it seriously.",
```
Should be removed as we can't guarantee that any whitecough is spreading when this appears.

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/dislike/decrease.json:
```diff
@@ -1,109 +1,151 @@
+			"m_c realizes that {PRONOUN/m_c/subject} misunderstood something r_c said.",
+			"m_c is too tired to be angry at r_c today, and learns something good about {PRONOUN/r_c/object} in the process.",
+			"m_c stops and thinks before assuming the worst of r_c.",
+			"When getting something to eat, m_c sees that r_c left behind {PRONOUN/r_c/poss} favorite prey for someone else to take.",
```
```suggestion
			"When getting something to eat, m_c sees r_c allow someone else to take {PRONOUN/r_c/poss} favorite prey",

```
Slight rewording to clear up ambiguity.  Previous version could be misread as r_c leaving m_c's favorite prey behind for someone else (not m_c) to take, which is opposite from the intention.

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/dislike/decrease.json:
```diff
@@ -1,109 +1,151 @@
+		"id": "dislike_de_med1",
+		"interactions": [
+			"m_c realized that {PRONOUN/m_c/subject} {VERB/m_c/were/was} too harsh in {PRONOUN/m_c/poss} judgement of r_c.",
+			"m_c doesn't remember why they started disliking r_c.",
```
```suggestion
			"m_c doesn't remember why {PRONOUN/m_c/subject} started disliking r_c.",

```

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/dislike/decrease.json:
```diff
@@ -1,109 +1,151 @@
+		"interactions": [
+			"m_c is surprised to hear r_c express an ideal {PRONOUN/m_c/subject} can agree with.",
+			"m_c didn't realize how generous r_c could be.",
+			"m_c sees r_c stand up for a Clanmate despite the pushback from their peers.",
```
```suggestion
			"m_c sees r_c stand up for a Clanmate despite the pushback from {PRONOUN/r_c/poss} peers.",

```

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/dislike/decrease.json:
```diff
@@ -1,109 +1,151 @@
+		"id": "dislike_de_high3",
+		"intensity": "high",
+		"interactions": [
+			"m_c and r_c heckled another Clan at the gathering together.",
```
```suggestion
			"m_c and r_c heckled another Clan at the Gathering together.",

```

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/dislike/decrease.json:
```diff
@@ -1,109 +1,151 @@
+			"m_c and r_c worked well together on a patrol.",
+			"When they return from a patrol, m_c and r_c seem more at peace with each other than usual.",
+			"After a patrol gone wrong, m_c helps an injured r_c back to camp.",
+			"m_c and r_c defended their Clan at the gathering together.",
```
```suggestion
			"m_c and r_c defended their Clan at the Gathering together.",

```

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/dislike/decrease.json:
```diff
@@ -1,109 +1,151 @@
+			"m_c and r_c heckled another Clan at the gathering together.",
+			"m_c and r_c worked well together on a patrol.",
+			"When they return from a patrol, m_c and r_c seem more at peace with each other than usual.",
+			"After a patrol gone wrong, m_c helps an injured r_c back to camp.",
```
Would have to remove this, as we can't guarantee it happens after a cat is injured.

scribblecrumb reviewed (changes requested):
Something to look out for in general is what interactions kittens can take part in.  For example, "m_c called out r_c for sleeping through a meeting" would be pretty silly if r_c was a kitten, but "r_c tries to jokingly convince m_c that an obvious lie is real." would be great if either cat was a kitten or any other status.  However, both of those interactions are contained in the same block and don't utilize any status constraints.  I'd suggest reading through your interactions and separating things out as necessary.

Certainly not finished reviewing haha, but I'm gonna take this in chunks since it's such a large addition.  Love all the new interactions so far though!!

howlidae commented:
Thank you for all your comments! I'm currently going through and giving another thorough read-over to prune out the stuff that doesn't make sense and any other errors I haven't caught. I'll reupload them all for further checks soon.

howlidae committed:
replaced files

 trimmed some weird interactions + fixed errors

howlidae commented:
Reuploaded!! Implemented the review feedback + applied the feedback to the rest of the files as well. Please let me know if I missed anything.

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/admiration/decrease.json:
```diff
@@ -15,7 +15,7 @@
 			"r_c tells m_c to smile more.",
 			"m_c had a funny dream about r_c and can't take {PRONOUN/r_c/object} seriously now.",
-			"r_c is talking to m_c, but {PRONOUN/m_c/subject} can't stop noticing the mud on {PRONOUN/r_c/poss} pelt."
+			"r_c is talking to m_c, but {PRONOUN/m_c/subject} can't stop noticing the mud on r'c pelt."
```
```suggestion
			"r_c is talking to m_c, but {PRONOUN/m_c/subject} can't stop noticing the mud on r_c's pelt."

```

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/jealousy/increase.json:
```diff
@@ -1,196 +1,198 @@
+			"m_c wishes {PRONOUN/m_c/subject} could get {PRONOUN/m_c/poss} pelt to shine like r_c's.",
+			"m_c notices one of {PRONOUN/m_c/poss} friends laughing with r_c.",
+			"m_c wants to asks a Clanmate to eat with {PRONOUN/m_c/object}, but they're busy with r_c.",
+			"A Clanmate that m_c wanted to be get to know better is already friends with r_c.",
```
```suggestion
			"A Clanmate that m_c wanted to get to know better is already friends with r_c.",

```

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/jealousy/increase.json:
```diff
@@ -1,196 +1,198 @@
+			"The Clan likes one of r_c's suggestions, but it was just m_c's reworded.",
+			"m_c tries to do something like r_c, but just ends up getting embarrassed.",
+			"A Clanmate accuses m_c of copying one of r_c's jokes.",
+			"m_c wishes {PRONOUN/m_c/subject} {VERB/m_c/look/looks} more like r_c.",
```
```suggestion
			"m_c wishes {PRONOUN/m_c/subject} looked more like r_c.",

```

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/jealousy/increase.json:
```diff
@@ -1,196 +1,198 @@
+	{
+		"id": "jealousy_inc_med3",
+		"interactions": [
+			"m_c is jealous that r_c got praised by the leader.",
```
```suggestion
			"m_c is jealous that r_c was praised by the leader.",

```

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/neutral.json:
```diff
@@ -1,154 +1,192 @@
+			"r_c asks m_c to speak up while chatting together.",
+			"m_c overthinks something {PRONOUN/m_c/subject} said to r_c.",
+			"m_c does something embarrassing in front of r_c, but r_c doesn't seem to mind.",
+			"m_c apologies to r_c, but r_c doesn't know why.",
```
```suggestion
			"m_c apologizes to r_c, but r_c doesn't know why.",

```

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/romantic/decrease.json:
There seems to be some interactions in this file that you removed.  Were they moved elsewhere or is this a mistake?

howlidae replied:
I haven't made any changes to the romantic decrease or increase file, so those are actually just old official versions of the files that I mistakenly uploaded when importing my edited interaction files. Sorry about that! I'll remove those now. I'm a bit of a grandma with GitHub.

howlidae committed:
Update resources/dicts/relationship_events/normal_interactions/admiration/decrease.json

fix typo

Co-authored-by: scribblecrumb <115410010+scribblecrumb@users.noreply.github.com>

howlidae committed:
Update resources/dicts/relationship_events/normal_interactions/jealousy/increase.json

fix typo

Co-authored-by: scribblecrumb <115410010+scribblecrumb@users.noreply.github.com>

howlidae committed:
Update resources/dicts/relationship_events/normal_interactions/jealousy/increase.json

fix typo

Co-authored-by: scribblecrumb <115410010+scribblecrumb@users.noreply.github.com>

howlidae committed:
Update resources/dicts/relationship_events/normal_interactions/jealousy/increase.json

fix typo

Co-authored-by: scribblecrumb <115410010+scribblecrumb@users.noreply.github.com>

howlidae committed:
Update resources/dicts/relationship_events/normal_interactions/neutral.json

fix typo

Co-authored-by: scribblecrumb <115410010+scribblecrumb@users.noreply.github.com>

howlidae committed:
fixed romantic decrease from outdated file

howlidae committed:
fix romantic increase file from outdated

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/admiration/increase.json:
```diff
@@ -1,311 +1,398 @@
+			"m_c is motivated by r_c's big smile.",
+			"r_c's good morning to m_c meant a lot to {PRONOUN/m_c/object}.",
+			"When tucking in for the night, m_c saw that r_c fixed {PRONOUN/m_c/poss} nest.",
+			"After the gathering, m_c notices that r_c is making sure everyone got home.",
```
```suggestion
			"After the Gathering, m_c notices that r_c is making sure everyone got home.",

```

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/admiration/increase.json:
```diff
@@ -1,311 +1,398 @@
+			"m_c and r_c find out they have the same favorite prey!",
+			"m_c and r_c come up with a catchy song together.",
+			"m_c listens to r_c's big plans for the day.",
+			"r_c points out to m_c that a spider is in {PRONOUN/m_c/poss} nest before {PRONOUN/m_c/subject} sits in it.",
```
```suggestion
			"r_c points out to m_c that a spider is in {PRONOUN/m_c/poss} nest before {PRONOUN/m_c/subject} {VERB/m_c/sit/sits} in it.",

```

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/admiration/increase.json:
```diff
@@ -1,311 +1,398 @@
+			"m_c wants to hold an important position like r_c.",
+			"m_c dreams about being like m_c in the future.",
+			"m_c works extra hard after seeing how skilled r_c is.",
+			"m_c tells {PRONOUN/m_c/poss} friend that r_c is their hero.",
```
```suggestion
			"m_c tells {PRONOUN/m_c/poss} friend that r_c is {PRONOUN/m_c/poss} hero.",

```

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/admiration/increase.json:
```diff
@@ -1,311 +1,398 @@
+			"m_c notices that r_c is always one of the first to volunteer for a difficult task.",
+			"During a Clan debate, m_c notices how closely r_c sticks to {PRONOUN/r_c/poss} beliefs.",
+			"Some things have been rough for r_c lately, but m_c admires {PRONOUN/r_c/poss} willingness to go on.",
+			"During rough weather, r_c helps m_c get safely to their den.",
```
```suggestion
			"During rough weather, r_c helps m_c get safely to {PRONOUN/m_c/poss} den.",

```

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/admiration/increase.json:
```diff
@@ -1,311 +1,398 @@
+			"m_c and r_c debate about something interesting. Though they disagree, they stay amicable.",
+			"m_c and r_c tell increasingly scary stories to one another.",
+			"m_c and r_c repair some nests together.",
+			"m_c compliments r_c on their bravery today.",
```
```suggestion
			"m_c compliments r_c on their {PRONOUN/r_c/poss} today.",

```

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/admiration/increase.json:
```diff
@@ -1,311 +1,398 @@
+			"m_c and r_c tell increasingly scary stories to one another.",
+			"m_c and r_c repair some nests together.",
+			"m_c compliments r_c on their bravery today.",
+			"Upon hearing a yowl at the edge of camp, m_c and r_c are the first on the scene. They don't find anyone there.",
```
hmmm not entirely sure if this makes for a good relationship event?  It feels more like a nice setup/prompt for a mystery, and would probably make more sense as a normal moon event rather than a relationship event.

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/admiration/increase.json:
```diff
@@ -1,311 +1,398 @@
+			"m_c has a big smile whenever r_c comes around.",
+			"m_c believes everything r_c says.",
+			"m_c tries to tell jokes as funny as r_c's, but they make little sense.",
+			"r_c lets m_c make a wish on a shooting star, even though {PRONOUN/m_c/subject} saw it first.",
```
```suggestion
			"r_c lets m_c make a wish on a shooting star, even though {PRONOUN/r_c/subject} saw it first.",

```

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/admiration/increase.json:
```diff
@@ -1,311 +1,398 @@
+		]
+	},
+	{
+		"id": "admeration_kit_inc_med2",
```
```suggestion
		"id": "admiration_kit_inc_med2",

```

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/admiration/increase.json:
```diff
@@ -1,311 +1,398 @@
+			"m_c is impressed by r_c's competence in any situation.",
+			"m_c is amazed at how skilled r_c really is.",
+			"m_c thinks the Clan's ancestors smile upon r_c.",
+			"m_c sees r_c save someone.",
```
Remove this I think, we already have injury/death moon events specifically geared towards "savior" scenarios.

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/admiration/increase.json:
```diff
@@ -1,311 +1,398 @@
+			"m_c is amazed at how skilled r_c really is.",
+			"m_c thinks the Clan's ancestors smile upon r_c.",
+			"m_c sees r_c save someone.",
+			"r_c sits in the sunrise, a halo of light around them. m_c takes it as a sign to admire {PRONOUN/r_c/object}.",
```
```suggestion
			"r_c sits in the sunrise, a halo of light around {PRONOUN/r_c/object}. m_c takes it as a sign to admire {PRONOUN/r_c/object}.",

```

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/admiration/increase.json:
```diff
@@ -1,311 +1,398 @@
+			"m_c thinks the Clan's ancestors smile upon r_c.",
+			"m_c sees r_c save someone.",
+			"r_c sits in the sunrise, a halo of light around them. m_c takes it as a sign to admire {PRONOUN/r_c/object}.",
+			"r_c gets hurt on a patrol but makes sure m_c's sickness is treated first.",
```
Remove, can't guarantee it coincides with an actual injury or sickness.

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/comfortable/decrease.json:
```diff
@@ -1,192 +1,263 @@
+			"m_c struggles to make sense of something trivial r_c said.",
+			"m_c can't help but feel uneased at r_c's words lately.",
+			"m_c noticed that r_c had a sour expression while looking at a Clanmate in the distance.",
+			"m_c felt eyes on their back, but when {PRONOUN/m_c/subject} looked, r_c was quickly turning away.",
```
```suggestion
			"m_c felt eyes on {PRONOUN/m_c/poss} back, but when {PRONOUN/m_c/subject} looked, r_c was quickly turning away.",

```

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/comfortable/decrease.json:
```diff
@@ -1,192 +1,263 @@
+			"m_c tells {PRONOUN/m_c/poss} Clanmates that something's off about r_c.",
+			"The moon casts a shadow on r_c's face in a way that makes m_c uneasy.",
+			"For a moment, m_c thought r_c's voice sounded like someone else's.",
+			"After r_c finished {PRONOUN/r_c/poss} food, m_c notices {PRONOUN/r_c/subject} made a mess."
```
```suggestion
			"After r_c finished {PRONOUN/r_c/poss} food, m_c noticed {PRONOUN/r_c/subject} made a mess."

```

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/comfortable/decrease.json:
```diff
@@ -1,192 +1,263 @@
+			"r_c talks to m_c a bit too enthusiastically about how best to kill prey.",
+			"r_c started a fight with m_c.",
+			"m_c was startled after r_c snarled viciously during a trivial disagreement.",
+			"m_c notices that r_c hasn't groomed fresh kill blood off {PRONOUN/r_c/poss} paw.",
```
```suggestion
			"m_c notices that r_c hasn't groomed fresh-kill blood off {PRONOUN/r_c/poss} paw.",

```

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/comfortable/decrease.json:
```diff
@@ -1,192 +1,263 @@
+			"m_c told r_c to shut up.",
+			"During a conversation with r_c, m_c says something too bluntly.",
+			"m_c calls out r_c for leaving old, rotten prey in the pile.",
+			"r_c thinks m_c is lying when {PRONOUN/r_c/subject} {VERB/r_c/apologize/apologizes}.",
```
```suggestion
			"r_c thinks m_c is lying when {PRONOUN/m_c/subject} {VERB/m_c/apologize/apologizes}.",

```

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/comfortable/decrease.json:
```diff
@@ -1,192 +1,263 @@
+			"m_c swore {PRONOUN/m_c/subject} saw r_c burying something.",
+			"m_c can smell blood on r_c, but there's nothing there.",
+			"m_c sees a darkness in r_c's eyes.",
+			"A heavy feeling of guilt weights on m_c whenever {PRONOUN/m_c/subject}{VERB/m_c/'re/'s} kind to r_c.",
```
```suggestion
			"A heavy feeling of guilt weighs on m_c whenever {PRONOUN/m_c/subject}{VERB/m_c/'re/'s} kind to r_c.",

```

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/comfortable/decrease.json:
```diff
@@ -1,192 +1,263 @@
+			"m_c sees a darkness in r_c's eyes.",
+			"A heavy feeling of guilt weights on m_c whenever {PRONOUN/m_c/subject}{VERB/m_c/'re/'s} kind to r_c.",
+			"The bad dreams that m_c has about r_c are progressively getting worse and worse.",
+			"Why does m_c always see r_c out at night?"
```
This isn't quite formatted appropriately for a rel event, though I see the intention!  "m_c wonders why they always see r_c out at night." could be a better wording.  Feel free to rework this one or remove it.

scribblecrumb reviewed (changes requested):
Some cool additions to comfort decrease!  Some of these do feel like they'd benefit from dislike increases occurring alongside them or existing relationship constraints of higher dislike.  Feel free to read back through them and add that where you feel it's appropriate, but if you'd rather not then no worries, I'm not gonna require it of you.

SunlitFable commented on resources/dicts/relationship_events/normal_interactions/trust/decrease.json:
```diff
@@ -1,207 +1,237 @@
+			"m_c can't seem to find r_c anywhere, and no one else has seen {PRONOUN/r_c/object} either.",
+			"m_c asks r_c a simple question, but r_c reacts strangely to it.",
+			"r_c forgot something that m_c asked {PRONOUN/r_c/object} to remember.",
+			"r_c won't stop looking at a specific Clanmate while m_c talks to {PRONOUN/r_c/object}.",
```
this might flow better with some altered wording
"r_c won't stop looking at a specific Clanmate while talking to m_c." for example!

SunlitFable commented on resources/dicts/relationship_events/normal_interactions/trust/decrease.json:
```diff
@@ -1,207 +1,237 @@
+			"m_c smells something strange on r_c.",
+			"m_c feels like r_c has been quieter than usual.",
+			"m_c noticed that r_c wasn't in {PRONOUN/r_c/poss} nest last night.",
+			"m_c can't seem to find r_c anywhere, and no one else has seen {PRONOUN/r_c/object} either.",
```
sentence feels a bit clunky hmm
may benefit from rewording. something like "Though m_c asks around, {PRONOUN/m_c/subject} can't find r_c anywhere."

SunlitFable commented on resources/dicts/relationship_events/normal_interactions/trust/decrease.json:
```diff
@@ -1,207 +1,237 @@
+			"r_c snitches on m_c for something minor.",
+			"m_c saw r_c hide something before talking to {PRONOUN/m_c/object}.",
+			"r_c won't tell m_c what {PRONOUN/r_c/subject} ate this morning.",
+			"r_c gets a strange expression when m_c walks up to ask a question.",
```
"gets a strange expression"
should be reworded
"has a strange look on their face" ?

SunlitFable commented on resources/dicts/relationship_events/normal_interactions/trust/decrease.json:
```diff
@@ -1,207 +1,237 @@
+			"r_c won't tell m_c what {PRONOUN/r_c/subject} ate this morning.",
+			"r_c gets a strange expression when m_c walks up to ask a question.",
+			"m_c has to snap r_c out of a daze to ask a question.",
+			"m_c subtly accuses r_c of being untruthful about something.",
```
"not being honest" may flow better than "being untruthful"

howlidae replied:
ended up removing it since i wasn't a fan of it ^^;

SnapshotManor commented on resources/dicts/relationship_events/normal_interactions/platonic/decrease.json:
Hello! I'm just peaking at this file.

Are you aware that some interactions are missing? For example, in ID "platonic_de_low1", the event "m_c noticed something r_c does that annoyed {PRONOUN/m_c/object}" is completely missing.

Was this intentional? We're just wondering if you were rewriting some vague ones or if you just accidentally removed some. Thank you <3

howlidae replied:
A little bit of both!! Some of them I moved from one file to another, some of them I rewrote for clarity or split into different interactions, and some of them I most likely removed on accident while I was going back to clean up lines according to Scribble's original advice. I can cross-reference and make sure that lines I deleted are present in one way or another!!

SunlitFable commented on resources/dicts/relationship_events/normal_interactions/trust/decrease.json:
```diff
@@ -1,207 +1,237 @@
+			"r_c sides with a Clanmate instead of m_c on something.",
+			"m_c thinks that r_c spread a rumor about {PRONOUN/m_c/object}.",
+			"m_c doesn't think that r_c's intentions are all pure.",
+			"m_c keeps thinking {PRONOUN/m_c/subject} {VERB/m_c/see/sees} r_c sneaking around the camp at night.",
```
"keeps thinking" may flow better as something like "swears"

SunlitFable commented on resources/dicts/relationship_events/normal_interactions/trust/decrease.json:
```diff
@@ -1,207 +1,237 @@
+			"m_c keeps thinking {PRONOUN/m_c/subject} {VERB/m_c/see/sees} r_c sneaking around the camp at night.",
+			"r_c keeps making excuses about something m_c wants to know.",
+			"m_c can't seem to get a straight answer from r_c lately.",
+			"r_c let m_c down while m_c was counting on {PRONOUN/r_c/object}."
```
"while" should be "when"

SunlitFable commented on resources/dicts/relationship_events/normal_interactions/trust/decrease.json:
```diff
@@ -1,207 +1,237 @@
+			"r_c keeps avoiding eye contact with m_c.",
+			"m_c learns that r_c has been misleading {PRONOUN/m_c/object} about something trivial.",
+			"r_c does something that badly affects m_c.",
+			"r_c agrees to something for m_c that {PRONOUN/m_c/object} didn't want to do...",
```
this one feels a little odd. why is trust decreasing here? did they agree to do something together, did m_c coerce them into it?

howlidae replied:
Rewrote this to be a little more clear:

r_c volunteers m_c for something that {PRONOUN/m_c/object} didn't want to do...

Is this vague enough to be for any age or rank of cat?

SunlitFable replied:
perfect! i'd say its vague enough, yes :3

SunlitFable commented on resources/dicts/relationship_events/normal_interactions/trust/decrease.json:
```diff
@@ -1,207 +1,237 @@
+			"m_c realizes that r_c's been avoiding {PRONOUN/m_c/object}.",
+			"m_c knows that r_c is hiding something.",
+			"m_c was convinced that r_c was lying about something, but r_c wasn't.",
+			"The Clan realizes that m_c doesn't ask for help from r_c much lately."
```
maybe instead of the entire clan, could have it be a Clanmate asking m_c why?

howlidae replied:
ended up removing this since I liked it in concept but it read pretty clunky no matter how I reworded it!

SnapshotManor commented on resources/dicts/relationship_events/normal_interactions/platonic/decrease.json:
```diff
@@ -1,194 +1,244 @@
+			"m_c has drawn the ire of r_c by deliberately taking the last of {PRONOUN/r_c/poss} favorite nesting materials for {PRONOUN/m_c/self}.",
+			"m_c and r_c keep talking over each other on accident.",
+			"After m_c cleans up an area, r_c accidentally messes it up again.",
+			"m_c tells something to r_c, but realizes r_c can't relate at all.",
```
Purrrsonally would prefer this to be "m_c tells something to r_c, but realizes {PRONOUN/r_c/subject} can't relate at all."

SunlitFable commented on resources/dicts/relationship_events/normal_interactions/trust/decrease.json:
```diff
@@ -1,207 +1,237 @@
+			"m_c tries to scare r_c.",
+			"m_c accuses r_c of being in cahoots with a cat from another Clan.",
+			"m_c publicly accuses r_c of breaking the Warrior Code.",
+			"m_c is blindsighted by r_c completely betraying {PRONOUN/m_c/object}.",
```
"blindsighted" should be "blindsided"

SunlitFable commented on resources/dicts/relationship_events/normal_interactions/trust/decrease.json:
```diff
@@ -1,207 +1,237 @@
+			"r_c's prank on m_c almost turned deadly.",
+			"r_c has been pitting a Clanmate against m_c.",
+			"m_c claims to have had a prophetic dream about r_c betraying the Clan.",
+			"m_c and r_c tell everyone embarrassing secrets about the other.",
```
"the other" may flow better as "each other"

SunlitFable commented on resources/dicts/relationship_events/normal_interactions/trust/decrease.json:
```diff
@@ -1,207 +1,237 @@
+		"id": "trust_for_medcat_de_high1",
+		"intensity": "high",
+		"interactions": [
+			"m_c can never seem explain {PRONOUN/r_c/poss} health in a way that makes r_c listen...",
```
i know this one's base-game, but "seem explain" should be "seem to explain"

SnapshotManor commented on resources/dicts/relationship_events/normal_interactions/platonic/decrease.json:
```diff
@@ -1,194 +1,244 @@
+			"m_c keeps asking r_c to explain a joke, but r_c won't do it.",
+			"m_c keeps asking r_c to explain a joke, but r_c just says that m_c had to be there.",
+			"m_c stretched in front of r_c. r_c didn't get a great view.",
+			"r_c asked m_c to cover {PRONOUN/m_c/poss} mouth while {PRONOUN/m_c/subject} cough, so m_c coughed on {PRONOUN/r_c/object} out of spite."
```
This needs to be verb tagged!
r_c asked m_c to cover {PRONOUN/m_c/poss} mouth while {PRONOUN/m_c/subject} {VERB/m_c/cough/coughs}, so m_c coughed on {PRONOUN/r_c/object} out of spite.

SunlitFable commented on resources/dicts/relationship_events/normal_interactions/trust/decrease.json:
```diff
@@ -1,207 +1,237 @@
+			"m_c accuses r_c of not taking care of cats properly.",
+			"m_c doesn't bother to ask r_c for help with a minor injury, worried that {PRONOUN/r_c/subject} might make it worse.",
+			"m_c complains that r_c doesn't seem to understand signs from StarClan very well.",
+			"m_c feels worse after r_c gave {PRONOUN/m_c/object} the wrong treatment for something minor."
```
this should either be "the wrong treatment" _or_ "the treatment for something else", not both

howlidae replied:
Rewrote to this for simplicity:

m_c thinks that r_c gave {PRONOUN/m_c/object} the wrong herbs.

SunlitFable commented on resources/dicts/relationship_events/normal_interactions/trust/decrease.json:
```diff
@@ -1,207 +1,237 @@
+			"vengeful"
+		]
+	}
+]
```
throwing this at the bottom; there are a lot of missing lines in here that haven't been rewritten. what is the reasoning for this? /genq

howlidae replied:
Copy-pasting my response from a few posts above!

Some of them I moved from one file to another, some of them I rewrote for clarity or split into different interactions, and some of them I most likely removed on accident while I was going back to clean up lines according to Scribble's original advice. I can cross-reference and make sure that lines I deleted are present in one way or another!!

Sorry about that!!!

SunlitFable replied:
all good! we weren't sure if it was intentional or not, so figured it was good to check :]

SunlitFable commented on resources/dicts/relationship_events/normal_interactions/trust/increase.json:
```diff
@@ -1,349 +1,400 @@
+			"m_c thinks about how r_c is always reliable.",
+			"m_c notices how r_c is being helpful around camp.",
+			"m_c has faith in r_c.",
+			"m_c notices that r_c has kept a secret of {PRONOUN/m_c/poss}s.",
```
"notices" may flow better as "is grateful"

SunlitFable commented on resources/dicts/relationship_events/normal_interactions/trust/increase.json:
```diff
@@ -1,349 +1,400 @@
+			"r_c explained something in detail to m_c that others wouldn't explain to {PRONOUN/m_c/object}.",
+			"m_c overhears r_c dispelling a false rumor about a Clanmate.",
+			"r_c helps m_c fix a matted piece of fur on {PRONOUN/m_c/poss} back.",
+			"m_c finds a Twoleg trinket. r_c agrees to keep it a secret.",
```
this would benefit from a bit more detail! do they bring it back to camp? are they playing with it? just finding a trinket by itself isn't a huge deal after all

howlidae replied:
Rewrote to this:
"m_c and r_c are playing with a fun Twoleg trinket they found.",

+ going to move it to a different file! Probably platonic increase.

SunlitFable commented on resources/dicts/relationship_events/normal_interactions/trust/increase.json:
```diff
@@ -1,349 +1,400 @@
+			"r_c doesn't entertain a rumor that is being spread about m_c.",
+			"r_c takes m_c's side while m_c is in a debate.",
+			"m_c nearly tripped, but was caught by r_c.",
+			"m_c asks r_c to watch {PRONOUN/m_c/poss} fresh-kill for {PRONOUN/m_c/object}.",
```
not strictly necessary, but could benefit from some more detail about _why_. is m_c going to the dirtplace?
this could also easily have a negative version!

SunlitFable commented on resources/dicts/relationship_events/normal_interactions/trust/increase.json:
```diff
@@ -1,349 +1,400 @@
+		"id": "trust_inc_med4",
+		"interactions": [
+			"r_c is building up m_c's confidence.",
+			"r_c guarantees to m_c that everything will be okay."
```
"guarantees to" may flow better as "promises"

SunlitFable commented on resources/dicts/relationship_events/normal_interactions/trust/increase.json:
```diff
@@ -1,349 +1,400 @@
+		"intensity": "high",
+		"interactions": [
+			"m_c tells r_c about {PRONOUN/m_c/poss} last encounter with StarClan.",
+			"m_c debates about the meaning of a prophetic dream with r_c.",
```
"about" should be removed from this sentence

SunlitFable commented on resources/dicts/relationship_events/normal_interactions/trust/increase.json:
```diff
@@ -1,349 +1,400 @@
+		"interactions": [
+			"m_c tells r_c about {PRONOUN/m_c/poss} last encounter with StarClan.",
+			"m_c debates about the meaning of a prophetic dream with r_c.",
+			"r_c puts a lot of faith into m_c's skill as {PRONOUN/m_c/poss} rank."
```
this feels a bit clunky. I'd personally recommend removing the part about their rank, or having separate versions for medicine cats and leaders

SunlitFable commented on resources/dicts/relationship_events/normal_interactions/trust/increase.json:
```diff
@@ -1,349 +1,400 @@
+		],
+		"random_status_constraint": [
+			"deputy",
+			"elder",
```
it's important to remember for a lot of interactions throughout this PR that elders can be as young as 6 moons old. personally for social inclusions of elders, i would recommend making it an age constraint instead, though admittedly i'm not too sure how that would work in tandem with status constraints. mayhaps worth considering sections for cats who are both seniors and elders

howlidae replied:
I forgot that elders can be so young!! The original file just had the line:
"m_c talks with r_c about difficult decisions."
so I just wanted to spruce it up a little. I think I'll just remove the ones I added and put them into a new section for leaders, medicine cats, and deputies.

SunlitFable commented on resources/dicts/relationship_events/normal_interactions/trust/increase.json:
```diff
@@ -1,349 +1,400 @@
+			"medicine cat"
+		]
+	}
+]
```
same as the prior file— would like to know the reasoning for removed interactions :]

howlidae committed:
Update resources/dicts/relationship_events/normal_interactions/admiration/increase.json

Co-authored-by: scribblecrumb <115410010+scribblecrumb@users.noreply.github.com>

howlidae committed:
Update resources/dicts/relationship_events/normal_interactions/admiration/increase.json

Co-authored-by: scribblecrumb <115410010+scribblecrumb@users.noreply.github.com>

howlidae committed:
Update resources/dicts/relationship_events/normal_interactions/admiration/increase.json

Co-authored-by: scribblecrumb <115410010+scribblecrumb@users.noreply.github.com>

howlidae committed:
Update resources/dicts/relationship_events/normal_interactions/admiration/increase.json

Co-authored-by: scribblecrumb <115410010+scribblecrumb@users.noreply.github.com>

howlidae committed:
Update resources/dicts/relationship_events/normal_interactions/admiration/increase.json

Co-authored-by: scribblecrumb <115410010+scribblecrumb@users.noreply.github.com>

howlidae committed:
Update resources/dicts/relationship_events/normal_interactions/admiration/increase.json

Co-authored-by: scribblecrumb <115410010+scribblecrumb@users.noreply.github.com>

howlidae committed:
Update resources/dicts/relationship_events/normal_interactions/admiration/increase.json

Co-authored-by: scribblecrumb <115410010+scribblecrumb@users.noreply.github.com>

howlidae committed:
Update resources/dicts/relationship_events/normal_interactions/comfortable/decrease.json

Co-authored-by: scribblecrumb <115410010+scribblecrumb@users.noreply.github.com>

howlidae committed:
Update resources/dicts/relationship_events/normal_interactions/comfortable/decrease.json

Co-authored-by: scribblecrumb <115410010+scribblecrumb@users.noreply.github.com>

howlidae committed:
Update resources/dicts/relationship_events/normal_interactions/comfortable/decrease.json

Co-authored-by: scribblecrumb <115410010+scribblecrumb@users.noreply.github.com>

howlidae committed:
Update resources/dicts/relationship_events/normal_interactions/comfortable/decrease.json

Co-authored-by: scribblecrumb <115410010+scribblecrumb@users.noreply.github.com>

howlidae committed:
Update resources/dicts/relationship_events/normal_interactions/comfortable/decrease.json

Co-authored-by: scribblecrumb <115410010+scribblecrumb@users.noreply.github.com>

howlidae committed:
Update admiration increase.json

howlidae committed:
Update decrease.json

howlidae committed:
Update decrease.json

howlidae committed:
Update decrease.json

howlidae committed:
Update increase.json

howlidae commented:
Adding a to do note for myself and for y'all's reference:
- restore deleted interactions, or ensure they have apt replacements if they needed editing
- swap some lines from original files to files that would make more sense
- implement Discord feedback
- ensure elder interactions are not tied to old age
- double check to make sure interactions are vague but not too vague

If/when this PR is complete, I was planning on adding more trait-specific interactions and possibly throwing up another PR. That's dependent on whether I have Brain Juice so we'll get there when we get there.

I really appreciate all your help!! I'm so sorry for the _plethora_ of errors >.> as I said in the Discord server, this was originally just meant as a fun mod to challenge myself and I can only double-triple check so many times before missing things. I'll post another comment when I'm done with edits for a final look-over!

howlidae committed:
final edits to admiration decrease (hopefully)

howlidae commented:
Quick question. In admiration increase, there's a section for elder admiration increase that goes against the principle that elders can be as young as 6 moons:

		

> 	"m_c feels like {PRONOUN/m_c/subject}'ll never accomplish as much as r_c.",
> 			"m_c wonders if {PRONOUN/m_c/subject} can ever be as good as r_c.",
> 			"m_c strives to catch up to r_c's success.",
> 

These are vanilla. Should this section be deleted/repurposed, or...?

howlidae committed:
almost final edits to admiration increase

Please refer to this comment: https://github.com/ClanGenOfficial/clangen/pull/3014#issuecomment-2494496871

howlidae committed:
final edits to comfortable decrease

deleted some interactions to prevent overlap, holding onto some to possibly use for trait-specific interactions

howlidae committed:
final edits to comfortable increase

deleted a few interactions and tidied up others

arisaema12 commented:
> Quick question. In admiration increase, there's a section for elder admiration increase that goes against the principle that elders can be as young as 6 moons:
> 
> > "m_c feels like {PRONOUN/m_c/subject}'ll never accomplish as much as r_c.",
> > "m_c wonders if {PRONOUN/m_c/subject} can ever be as good as r_c.",
> > "m_c strives to catch up to r_c's success.",
> 
> These are vanilla. Should this section be deleted/repurposed, or...?

maybe we could add an age constraint to fix this? `"random_age_constraint": ["senior adult", "senior"]`

scribblecrumb commented:
Sounds like a perfect solution to me!

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/dislike/increase.json:
```diff
@@ -1,571 +1,435 @@
+			"Just because they were told to stop bickering doesn't mean m_c and r_c will stop leering at each other.",
+			"m_c didn't warn r_c about a rock on the ground, which caused r_c to trip.",
+			"m_c claimed to give r_c a piece of crowfood on accident.",
+			"m_c spent the day arguing with r_c whether {PRONOUN/r_c/subject} stole something from m_c.",
```
```suggestion
			"m_c spent the day arguing with r_c about whether {PRONOUN/r_c/subject} stole something from m_c.",

```

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/dislike/increase.json:
```diff
@@ -1,571 +1,435 @@
+			"m_c punishes r_c with extra work.",
+			"m_c divides r_c into extra patrols.",
+			"m_c gives r_c a particularly hard task, expecting {PRONOUN/r_c/object} to fail.",
+			"m_c threatens r_c with never passing {PRONOUN/r_c/poss} assessment!",
```
Needs to be in an apprentice specific interaction, won't make sense for a warrior.

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/dislike/increase.json:
```diff
@@ -1,571 +1,435 @@
+			"m_c divides r_c into extra patrols.",
+			"m_c gives r_c a particularly hard task, expecting {PRONOUN/r_c/object} to fail.",
+			"m_c threatens r_c with never passing {PRONOUN/r_c/poss} assessment!",
+			"m_c tells other cats that r_c is not doing well in {PRONOUN/r_c/poss} training.",
```
same here

howlidae replied:
This one and the one above it are vanilla, so I'll go ahead and put them into an apprentice constraint!!

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/dislike/increase.json:
```diff
@@ -1,571 +1,435 @@
+			"m_c threatens r_c with never passing {PRONOUN/r_c/poss} assessment!",
+			"m_c tells other cats that r_c is not doing well in {PRONOUN/r_c/poss} training.",
+			"m_c tells r_c that {PRONOUN/r_c/subject}'ll never be the leader.",
+			"m_c always tries to avoid making r_c the leader of a patrol.",
```
Won't be appropriate for an apprentice, needs to be warrior specific

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/dislike/increase.json:
```diff
@@ -1,571 +1,435 @@
+		"interactions": [
+			"m_c tried to have a friendly discussion with r_c about what prey tastes best, but it turned into an argument.",
+			"m_c and r_c always end up arguing over what games to play.",
+			"m_c tells older Clanmate that r_c did something bad.",
```
```suggestion
			"m_c tells an older Clanmate that r_c did something bad.",

```

scribblecrumb reviewed (changes requested):
Last file that needed reviewing!

Just wanted to say thanks for being so willing to address changes to this PR.  I hope it hasn't been too overwhelming ^^; I'm excited to hear that you're interested in making more PRs in the future!  Once this PR is merged, we can actually make you a dev in the discord if you would like, then you'll have access to the dev side of the server and will be able to discuss changes with the other writers.

howlidae committed:
Update resources/dicts/relationship_events/normal_interactions/dislike/increase.json

Co-authored-by: scribblecrumb <115410010+scribblecrumb@users.noreply.github.com>

howlidae committed:
final edits to dislike increase

restored deleted interactions, removed a few custom interactions, fixed typos, added another age constraint. There is some overlap between the leader med increase 1 and 2 to allow for interactions not tied to adult age to still occur

howlidae committed:
final edits to admiration increase

added age constraint to admire_adult_odd_med1

howlidae committed:
final edits to dislike decrease

deleted some interactions that were too specific, ensured all original interactions are present. Some of them were moved to different sections that made more sense. (i.e. "m_c spends some time with r_c...little better" was moved so that they both get dislike decreased)

howlidae committed:
final edits to jealousy decrease

clarified a few things and took out a line or two

howlidae committed:
final edits to jealousy increase

some typo fixes + removed a line or two

scribblecrumb commented:
Request a review from me whenever you've finished your changes <3

howlidae committed:
resubmitting jealousy increase

says it's an invalid json but i double triple checked + ran it through jsonlint and it says it's fine??

howlidae committed:
final edits to neutral

removed some interactions

howlidae committed:
final edits to platonic decrease

restored deleted interaction, changed around a few, deleted a couple

howlidae committed:
final edits to platonic increase

i think i only removed one interaction. this one's cute. shoutout to whoever wrote all the kit interactions

howlidae committed:
final edits to trust decrease

Fixed One(1) typo

howlidae committed:
final edits to trust increase

howlidae commented:
Everything should be done!! Please double check my work on jealousy increase.json in particular since it flagged as invalid. I can't figure out why >.>

scribblecrumb committed:
trailing comma fix

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/neutral.json:
```diff
@@ -13,29 +13,28 @@
 			"m_c points out a constellation to r_c.",
 			"m_c and r_c keep trying to figure out what a bad smell in camp is.",
-			"m_c poses a crazy theory to r_c and they chat about it for a bit.",
-			"The whole Clan sees m_c and r_c accidentally headbutt each other. Embarrassing.",
```
Looks like a few events got removed, I'll mark them for you.

howlidae replied:
Most events that got removed in this "neutral" file were because I wasn't a big fan of them whether it's because they didn't read very smoothly or just didn't really make sense for a neutral interaction!

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/neutral.json:
```diff
@@ -13,29 +13,28 @@
 			"r_c has to help m_c stop choking on a huge bone.",
 			"m_c and r_c chat idly about the latest meeting.",
 			"m_c and r_c keep making each other yawn.",
-			"m_c and r_c realize to their horror that there's a dead animal in the puddle they are drinking from.",
```
Removed

howlidae replied:
Thought this one was a bit too unpleasant. ^^;

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/neutral.json:
```diff
@@ -47,8 +46,7 @@
 			"m_c asks r_c to remind {PRONOUN/m_c/object} of who's already gone on patrol today.",
 			"m_c asks r_c about a certain rule of the Warrior Code that {PRONOUN/m_c/subject} forgot.",
 			"m_c watches r_c training for something and nods approvingly at {PRONOUN/r_c/poss} progress.",
-			"m_c should probably break up the scuffle between r_c and a Clanmate, but it's entertaining.",
```
Removed

howlidae replied:
Removed this to save it for a group interaction if I do those in the future :)!!

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/neutral.json:
```diff
@@ -73,13 +71,10 @@
 			"r_c asks m_c to speak up while chatting together.",
 			"m_c overthinks something {PRONOUN/m_c/subject} said to r_c.",
 			"m_c does something embarrassing in front of r_c, but r_c doesn't seem to mind.",
-			"m_c apologizes to r_c, but r_c doesn't know why.",
```
Removed

howlidae replied:
Moved this out of the "nervous" interactions to the general neutral interactions!

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/neutral.json:
```diff
@@ -89,17 +84,14 @@
 		"id": "neutral_interaction4",
 		"interactions": [
 			"m_c stands tall when r_c walks by.",
-			"m_c doesn't mind r_c staring.",
```
Removed

howlidae replied:
Thought this one sounded a bit creepy, so I removed it. LOL

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/neutral.json:
```diff
@@ -89,17 +84,14 @@
-			"r_c sees m_c try and fail to something after claiming {PRONOUN/m_c/subject} {VERB/m_c/were/was} an expert at it.",
+			"r_c watches m_c try and fail at something after claiming {PRONOUN/m_c/subject} {VERB/m_c/were/was} an expert at it.",
 			"m_c puffs out {PRONOUN/m_c/poss} chest when r_c approaches.",
-			"m_c flashes a confident smile at r_c, but r_c doesn't know why.",
```
Removed

howlidae replied:
Removed since I didn't think it made much sense! It overlaps a little with the other neutral confident interactions ^^

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/neutral.json:
```diff
@@ -89,17 +84,14 @@
 			"r_c doesn't bother to correct the blatantly wrong thing m_c said.",
 			"m_c does not take r_c's correction on something {PRONOUN/m_c/subject} said very well.",
 			"m_c falls flat on {PRONOUN/m_c/poss} face in front of r_c, but just gets right back up and keeps walking.",
-			"r_c compliments m_c about something, but m_c says {PRONOUN/m_c/subject} already knew that.",
```
Removed

howlidae replied:
Saving this one for a dislike increase interaction for "confident" trait ^^

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/neutral.json:
```diff
@@ -123,11 +115,9 @@
 			"r_c seems to be oblivious to m_c glaring at {PRONOUN/r_c/object} from across camp.",
 			"m_c is sharpening {PRONOUN/m_c/poss} claws near r_c.",
 			"r_c notices that m_c knows a lot about the Dark Forest.",
-			"m_c says a fun fact about the Dark Forest to r_c.",
```
Removed

howlidae replied:
Thought this one was too similar to the one right above it!

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/neutral.json:
```diff
@@ -123,11 +115,9 @@
 			"r_c wonders when m_c learned so much about cat anatomy.",
 			"r_c has to snap m_c out of staring darkly at another Clanmate.",
 			"r_c sees m_c using a small animal bone to pick food from {PRONOUN/m_c/poss} teeth.",
-			"As r_c passes by, {PRONOUN/r_c/subject} {VERB/r_c/smell/smells} blood on m_c's fur.",
```
Removed

howlidae replied:
This one is a bit too similar to one I wrote in comfort decrease, so I removed it

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/neutral.json:
```diff
@@ -157,11 +147,11 @@
 		"id": "neutral_interaction8",
 		"interactions": [
 			"r_c notices m_c sulking around camp again.",
-			"r_c keeps hearing m_c sighing sadly nearby.",
```
Removed

howlidae replied:
Didn't really liked how this one flowed <3

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/neutral.json:
```diff
@@ -177,17 +167,14 @@
 			"r_c thinks it's funny how closely m_c protects {PRONOUN/m_c/poss} fresh-kill.",
-			"m_c tries to say something rude, but r_c just laughs at it.",
-			"r_c isn't phased by m_c's grumpiness today.",
-			"m_c snaps at r_c, but r_c knows it's just how {PRONOUN/m_c/subject} {VERB/m_c/are/is}."
```
Removed

howlidae replied:
Thought this one overlapped with other interactions in the same block, or were less interesting ^^

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/platonic/decrease.json:
```diff
@@ -46,7 +45,6 @@
 			"r_c accuses m_c of lying about something m_c has no reason to lie about.",
 			"m_c is hurt by one of r_c's pranks, but r_c doesn't seem to believe {PRONOUN/m_c/object}.",
 			"m_c thinks that r_c is acting overly dramatic lately.",
-			"One of m_c's Clanmates tells {PRONOUN/m_c/object} about something crass r_c did.",
```
Removed

howlidae replied:
didn't flow very well so I removed it

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/platonic/increase.json:
```diff
@@ -45,7 +45,6 @@
 			"m_c thinks r_c has interesting things to talk about.",
 			"m_c is fascinated by one of r_c's stories.",
 			"m_c likes the way r_c laughs.",
-			"m_c thinks that r_c is good at leading others.",
```
Removed

howlidae replied:
thought it was a little vague, and it'd be a little funny for someone to think a 2 month old kit is good at leading others LOL

scribblecrumb commented on resources/dicts/relationship_events/normal_interactions/trust/increase.json:
```diff
@@ -22,8 +22,7 @@
 			"r_c makes a small sacrifice that benefits m_c.",
 			"r_c lets m_c know that a Clanmate was talking badly about {PRONOUN/m_c/object}.",
-			"r_c tells m_c that {PRONOUN/r_c/subject} was worried for {PRONOUN/m_c/object}.",
-			"r_c apologized to m_c for possibly hurting {PRONOUN/m_c/poss} feelings."
```
removed

howlidae replied:
Put these back into their respective trait constraint since I thought they worked there more!

scribblecrumb reviewed (changes requested):
Noticed a few lines that seemed to have been accidentally removed!  If there was a reasoning, just lmk.  Otherwise, it's all looking good!  I took care of the failing json for you, it was just a couple trailing commas.

howlidae commented:
> Noticed a few lines that seemed to have been accidentally removed! If there was a reasoning, just lmk. Otherwise, it's all looking good! I took care of the failing json for you, it was just a couple trailing commas.

Already put a few comments beneath the ones you've pointed out -- any ones that were removed were removed on purpose for one reason or another! You're free to reimplement them if you'd like, but I just thought a lot of them overlapped, didn't flow well, or were too specific/too vague ^^

scribblecrumb reviewed (approved):
Thanks for the explanations!  Gonna get this merged :> and just to double check, are you interested in receiving dev roles on the discord?  It does come with an expectation that you plan to continue contributing and you would be able to discuss ideas with the other writers on our team.

howlidae commented:
> Thanks for the explanations! Gonna get this merged :> and just to double check, are you interested in receiving dev roles on the discord? It does come with an expectation that you plan to continue contributing and you would be able to discuss ideas with the other writers on our team.

Yes, please!! I'd love that!! Thank you so much for your patience and support!!!
